### PR TITLE
Add support for multiwallet

### PR DESCRIPTION
**Pull request by ruimarinho**
Since Bitcoin Core v0.15.0, it's possible to manage multiple wallets using a single daemon. This enables use-cases such as managing a personal and a business wallet simultaneously in order to simplify accounting and accidental misuse of funds.

Historically, the accounts feature was supposed to offer similar functionality, but it has now been replaced by this more powerful feature.

This PR add support for multiple wallets and closes https://github.com/ruimarinho/bitcoin-core/issues/40.

### CODE REVIEW

**Comment by juscamarena** [comment]
Any reason this hasn't been merged yet?

I can help test if needed.

**Comment by ruimarinho** [comment]
@juscamarena none really, just missing a peer review from either @pedrobranco or @joaopaulofonseca. If you can help testing, that would be great help as well.

**Comment by pedrobranco** [comment]
@juscamarena Any feedback from this PR?

File: src/methods.js [hunk 1]
```diff
@@ -11,153 +11,619 @@ import { map, set } from 'lodash';
...
+  },
   walletPassphrase: {
+    category: 'wallet',
     obfuscate: {
       request: params => set([...params], '[0]', '******')
     },
     version: '>=0.1.0'
   },
-  walletPassphraseChange: { version: '>=0.1.0' }
+  walletPassphraseChange: {
```
Comment (pedrobranco) [inline src/methods.js]: Please obfuscate the request passwords (`params[0]` and `params[1]`).

**Comment by pedrobranco** [comment]
utACK. Will test it soon.

**Comment by kerematam** [comment]
Merging anytime soon?

File: src/methods.js (hunk 1 above)
Comment (RyuMaster) [reply src/methods.js]: Can I just open new merge request will all stuff fixes? Seems like original commit author is not there to do this.

**Comment by pedrobranco** [comment]
Rebased and applied some fixes:
- Missing `bitcoin-core:0.15.0.1-alpine` image on docker-compose: was replaced with `bitcoin-core:0.15-alpine`.
- Incompatible patch / minor releases of babel dev dependencies with `node@4`: fixed to the last supported version.

**Pull request merged**